sdk/ai/ai-projects/src/agents/files.ts

```diff
@@ -31,6 +31,19 @@ export async function uploadFile(
   return result.body; 
 }
 
+/** Uploads a file for use by other operations. */
+export async function uploadFileAndPoll(
```

Comment:
Would it be possible to make this an LRO operation? using @azure/core-lro.  /cc @joheredi @MaryGao

Reply:
@ganeshyb would we want to use an LRO here?

Reply:
I am not sure actually, could you help me understand what the traffic will look like? 

We leverage the first request to upload the file successfully but when this request is back this doesn't mean the server side is completed. So we have following requests to poll status? Will this be the traffic?

Reply:
Yes, The scenarios is first the user uploads the file, based on the file size and processing on the backend, its goes through states like pending, running, This api waits for the file to be uploaded and wait till its processed.

Reply:
Interesting, this seems to be an LRO operation for me. I have two questions here
- Does this operation follow our [LRO guideline](https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-operations)?
- Have we reviewed with this with our architects? Why the spec is still normal operation: https://github.com/Azure/azure-rest-api-specs/blob/0d2ca01a170641d8b1f5dace38b8995bb7563be8/specification/ai/Azure.AI.Projects/agents/files/routes.tsp#L48-L74?